src/languages/stan.js

```diff
@@ -504,16 +345,38 @@ export default function(hljs) {
       $pattern: hljs.IDENT_RE,
       title: BLOCKS,
       keyword: STATEMENTS.concat(VAR_TYPES).concat(SPECIAL_FUNCTIONS),
-      built_in: FUNCTIONS
+      built_in: FUNCTIONS.concat(DISTRIBUTIONS).concat(expanded),
+      punctuation: DELIMIT
     },
     contains: [
       hljs.C_LINE_COMMENT_MODE,
       INCLUDE,
       hljs.HASH_COMMENT_MODE,
       BLOCK_COMMENT,
+     {
+       className: 'built_in',
+       variants: [
+         {
+          begin: /(?<=~)\s+(\w+)/
+         },
+         {
+          begin: /(?<=\+=)\s+(\w+(_lpdf|_lupdf|_lpmf|_cdf|_lcdf|_lccdf|_qf))/
+         }
+        ] 
+     },
+     {
+      className: 'symbol',
+      begin: /\s+(pi|e|sqrt2|log2|log10)(?=\()/,
```

Comment:
`\s+` likely should not be part of the symbol match... we do not include outside space in scopes - we try to be precise like that.  If you need to match the space before you likely need a multi-matcher (passing an array of regex to `match`, see the docs).

Reply:
thanks, removed